apps/prairielearn/src/lib/assets.ts

```diff
@@ -138,6 +138,13 @@ export async function init() {
   });
 }
 
+/**
+ * Shuts down the development assets compiler if it is running.
+ */
+export async function close() {
+  await compiledAssets.close();
```

Comment:
```suggestion
  return compiledAssets.close();
```

No point in awaiting here instead of just returning the promise.

Reply:
I generally agree in performance-sensitive code (no point waiting for two promise resolutions), but this will only ever be run during termination of the dev server during tests, and I think it's clearer to have the explicit await here.